include/freshports.php

```diff
@@ -825,7 +825,7 @@ function freshports_Logo() {
 
     if (defined('SHOW_ANIMATED_BUG') && SHOW_ANIMATED_BUG)
     {
-	  $HTML .= '<img src="/images/notbug.gif" alt="notbug" title="notbug">';
+	  $HTML .= '<img src="/images/notbug.gif" width="56" height="50" alt="notbug" title="notbug">';
```

Comment:
From time to time, people complain about that bug. I've toyed with the idea of paid subscriptions where you don't see the bug. I've also considered IPv6 only, deciding randomly to display the bug, etc.